cranelift/filetests/src/test_run.rs

```diff
@@ -42,8 +42,22 @@ impl SubTest for TestRun {
                 let command = parse_run_command(trimmed_comment, &func.signature)
                     .map_err(|e| format!("{}", e))?;
                 trace!("Parsed run command: {}", command);
-                // TODO in following changes we will use the parsed command to alter FunctionRunner's behavior.
 
+                // If this test requests to run on a completely different
+                // architecture than the host platform then we skip it entirely,
+                // since we won't be able to natively execute machine code.
+                let requested_arch = context.isa.unwrap().triple().architecture;
+                if requested_arch != Architecture::host() {
+                    return Ok(());
+                }
```

Comment:
And we might want to print something saying we skipped a test.

Reply:
Does the filetest infrastructure print out anything else currently? Curious if I can see an example of what format to print and such.

Reply:
Over in `cranelift/src/filetests/runner.rs` it is printing `FAIL {path}: {error}` so maybe we could print `IGNORE {path}: host ISA did not match target ISA`? That way at least there is some indication that things didn't run perfectly.

Reply:
Hm so it looks like the `path` isn't already threaded through, would you be ok to follow-up with this at a later date?

Reply:
Sure, tag me in an issue and I will try to look at this later.

Reply:
Sure thing! Extracted out https://github.com/bytecodealliance/wasmtime/issues/1558